client/client_utils.cpp

```diff
@@ -2,66 +2,102 @@
 
 std::mutex fileMutex;
 
-Operation genRandOperation() {
-	float r = (float) rand() / RAND_MAX;
-	int key = rand() % KEY_MAX;
-  
-	Operation op;
-	op.__set_op(r < 0.5 ? OpType::PUT : OpType::GET);
-  op.__set_key(std::to_string(key));
-
-	std::string value;
-
-  // If operation is GET, then set value to random bytes
-  // If operation is PUT, then set value to random int value
-	if (op.op == OpType::GET) {
-		char rand_val[VALUE_SIZE];
-		randombytes_buf(rand_val, VALUE_SIZE);
-		value = std::string(rand_val);
-	} else {
-		int put_val = rand() % VAL_MAX;
-		value = std::to_string(put_val);
-	}
-	op.__set_value(clientEncrypt(value));
-
-	return op;
+void parseArgs(int argc, char *argv[], std::ifstream &seed, bool &init_db,
```

Comment:
Why don't we use an argument parsing library like [this](https://github.com/p-ranav/argparse)